akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/ActorContext.scala

```diff
@@ -95,12 +95,11 @@ trait ActorContext[T] {
 
   /**
    * Force the child Actor under the given name to terminate after it finishes
-   * processing its current message. Nothing happens if the ActorRef does not
-   * refer to a current child actor.
+   * processing its current message. Nothing happens if the ActorRef is a child that is already stopped.
    *
-   * @return whether the passed-in [[ActorRef]] points to a current child Actor
+   * @throws IllegalArgumentException if the given actor ref is not a direct child of this actor
```

Comment:
also clarify that it will not throw if the given child actor is already stopped, i.e. it's idempotent

Reply:
Sentence above says: `Nothing happens if the ActorRef is a child that is already stopped.`